lib/serialisers/json-0.6.js

```diff
@@ -317,7 +323,14 @@ module.exports = JSONSerialiser.extend({
     if (item.element === 'object') {
       // This is an object, so we need to check if it's members contain
       // additional information
-      // TODO
+      var values = [];
+      var content = item.content || [];
+
+      for (var index = 0; index < content.length; index++) {
+        values.push(this.serialise(content[index]));
+      }
+
+      return values;
```

Comment:
This could be simpler using map:

```js
return (item.content || []).map(this.serialise);
```

Possibly needs thisArg:

```js
return (item.content || []).map(this.serialise, this);
```

Reply:
I want to leave this as it is, because it might need some modifications later on. I am not sure this fixes all the cases that can arise.